Utils/Dataflow/run/data4es-start

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash -l
 
-START_OFFSET="09-05-2016 00:00:00"
 BATCH_SIZE=100
-DEBUG=
+DEBUG=1
 
 base_dir=$( cd "$(dirname "$(readlink -f "$0")")"; pwd)
 
+log() {
+  level=INFO
+  [ $# -eq 2 ] && level="$1" && shift
+  msg="$1"
+  echo "($level) `date +'%d-%m-%Y %T'` (data4es) $msg" >&2
```

Comment:
Since log() is a candidate for moving into a library, it could be useful to replace this `data4es` with a variable - a value passed to log(), or script name, or something else. Probably, not now and not in this PR.

Reply:
Good point; I decided not to hold it back for too long, so here it is (b5d6284).
Thank you!